01-syntax/05-oop/Sequence.js

```diff
@@ -1 +1,59 @@
-// → Your code here.
+function ArraySeq(arr) {
```

Comment:
You need additional `eslint` props to pass `npm run lint`  
Take a look [here](https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/05-oop/Sequence.js#L1)
And rename `ArraySeq` to `Sequence` (as a name of your module/file)